src/briefcase/integrations/docker.py

```diff
@@ -118,12 +96,108 @@ def verify_docker(command):
+        ).strip('\n')
+
+        # Do a simple check that the docker that was invoked
+        # actually looks like the real deal, and is a version that
+        # meets our requirements.
+        if output.startswith('Docker version '):
+            docker_version = output[15:]
+            version = docker_version.split('.')
+            if int(version[0]) < 19:
+                # Docker version isn't compatible.
+                raise BriefcaseCommandError(
+                    WRONG_DOCKER_VERSION_ERROR_MESSAGE.format(
+                        docker_version=docker_version,
+                        **docker_install_details(command.host_os),
+                    )
+                )
+
+        else:
+            print(UNKNOWN_DOCKER_VERSION_WARNING)
+    except subprocess.CalledProcessError:
+        print(DOCKER_INSTALLATION_STATUS_UNKNOWN_WARNING)
+    except FileNotFoundError:
+        # Docker executable doesn't exist.
+        raise BriefcaseCommandError(
+            DOCKER_NOT_INSTALLED_ERROR_MESSAGE.format(
+                **docker_install_details(command.host_os)
+            )
+        )
+
+    _verify_docker_can_run(command)
+    # will raise BriefcaseCommandError and print help message
+    # if a test docker command fails
 
     # Return the Docker wrapper
     return Docker
 
 
+def _verify_docker_can_run(command):
+    # Docker is probably installed, but it still
+    # may not have the right permissions to run
+
+    LACKS_PERMISSION_ERROR_MESSAGE = '''
+*************************************************************************
+** ERROR: docker command lacks relevant permissions                    **
+*************************************************************************
+
+docker reported an error when Briefcase attempted to use it.  It is
+possible that docker requires sudo privileges in its current config.
+
+See docs at https://docs.docker.com/engine/install/linux-postinstall/
+for information, or try running Briefcase build with the --no-docker
+option (note this may require installing additional dependencies)
+
+*************************************************************************'''
+    DAEMON_NOT_RUNNING_ERROR_MESSAGE = '''
+*************************************************************************
+** ERROR: docker daemon not running                                    **
+*************************************************************************
+
+Briefcase is unable to use docker commands because the docker daemon
+appears to not be running.
+
+See docs at https://docs.docker.com/config/daemon/ for help
+ensuring that it is running or debugging any issues, or try
+running Briefcase build with the --no-docker option
+(note this may require installing addtional dependencies)
+
+*************************************************************************'''
+    GENERIC_ERROR_MESSAGE = '''
+docker command failed with error: {error_message}'''
+
+    try:
+        _ = command.subprocess.check_output(
```

Comment:
Minor style convention - if the return value isn't being used, preference is to not assign the value at all, rather than assigning to a dummy variable.